fern/docs/pages/manual/nlsql.mdx

```diff
@@ -0,0 +1,30 @@
+## MySQL Configuration
+Text-to-SQL querying has been tested for [MySQL](https://www.mysql.com/) in PrivateGPT.
+
+To use the text to SQL feature, set the following environment variables to allow connecting to your MySQL Server:
+
+```yaml
+sqldatabase:
+  dialect: mysql
+  driver: pymysql
+  host: <db_hostname_here>
+  user: <db_username_here>
+  password: <db_password_here>
+  database: <database_name_here>
+  tables: <list_of_table_names_here>
+```
```

Comment:
this is very rigid, why not move it to a parameter similar to `context_files` (like `context_database(s)`)? Conceptually speaking there is not big difference between talking to your documents or talking to your database(s)

I understand that would require a big refactor of this PR but it will set a pattern for next features (like `context_websites`)